packages/next-sitemap/src/interface.ts

```diff
@@ -196,7 +196,13 @@ export interface IRoutesManifest {
   i18n?: {
     locales: string[]
     defaultLocale: string
-  }
+  },
+  dynamicRoutes: [{
+    [key: string]: string[]
+  }],
+  staticRoutes: [{
+    [key: string]: string[]
+  }]
```

Comment:
This seems to fix the build errors for me at least:
```suggestion
  dynamicRoutes?: {
    page: string
  }[]
  staticRoutes?: {
    page: string
  }[]
```